### DISCUSSION TITLE
Member Alignment for Connection Exported from Checkbot

### DISCUSSION THREAD

**Discussion by dr13377** [idea-statica/ideastatica-public#383] [General Discussion]
Hi,

I have exported a few connections from a checkbot model (created from SAF file import). The connections have been exported to "separate files" to allow manipulation of the connection. However, when the Align drop down is set to "To Member Plate" there are no options in the drop down box (see image below). 
![image](https://github.com/idea-statica/ideastatica-public/assets/91706141/0ad4c145-c721-4b9b-a20f-f2eb89da9a2b)

If I add a member to this connection then there are options to align the member to member plate. Is there another setting I need to change?
![image](https://github.com/idea-statica/ideastatica-public/assets/91706141/9e5eba4a-ea95-4f66-94fb-9444ac7bf9cd)

Thanks

**Comment by nathanlukeIDEA** [comment]
HI @dr13377, can you please confirm your version?

**Reply by dr13377** [reply]
Hi @nathanlukeIDEA, my product version is 23.1.0.4061.

**Reply by nathanlukeIDEA** [reply]
Hi @dr13377, it seems to be working on another connection I have imported from SAF and then exported using that version. Are you able to send your file to the support centre so they can take a look at it?

**Reply by dr13377** [reply]
Hi @nathanlukeIDEA. I have upgraded to the latest version and I am having fewer instances of this occurring however, it still seems to happen now and then. It happens more frequently when exporting multiple connections (many connections) from the checkbot. I will send my SAF file and an example of an exported connection that doesn't allow member alignment.

**Comment by nathanlukeIDEA** [comment]
Fixed in v23.1.3
